sample/src/main/res/values/styles.xml

```diff
@@ -8,6 +8,7 @@
         <item name="colorAccent">@color/colorAccent</item>
         <item name="colorControlNormal">@color/colorPrimary</item>
         <item name="android:windowBackground">@color/background</item>
+        <item name="floatingSearchViewStyle">@style/CustomFloatingSearchViewTheme</item>
```

Comment:
This change has to do with an error that is shown in the UI editor. It goes like:

```
Missing styles. Is the correct theme chosen for this layout?  Use the Theme combo box above the layout to choose a different layout, or fix the theme style references.  Failed to find style 'floatingSearchViewStyle' in current theme
```